supertokens_python/recipe/session/with_jwt/recipe_implementation.py

```diff
@@ -87,10 +93,20 @@ async def get_session(
         request: BaseRequest,
         anti_csrf_check: Union[bool, None],
         session_required: bool,
+        override_global_claim_validators: Optional[
+            Callable[
+                [List[SessionClaimValidator], SessionContainer, Dict[str, Any]],
+                MaybeAwaitable[List[SessionClaimValidator]],
+            ]
+        ],
```

Comment:
this argument is not needed in recipe implementation

Reply:
removed.